pkg/agent/agent_windows.go

```diff
@@ -160,6 +169,11 @@ func (i *Initializer) prepareOVSBridge() error {
 	if err = util.ConfigureInterfaceAddressWithDefaultGateway(brName, uplinkNetConfig.IP, uplinkNetConfig.Gateway); err != nil {
 		return err
 	}
+	// Restore host routes which are lost when moving network configuration of uplink interface to OVS bridge local interface.
```

Comment:
Restore the host routes
when moving the network configuration of the uplink interface to the OVS bridge interface.

Reply:
Done